### PR TITLE
Feature/headerbar

### PR DESCRIPTION
**Pull request by SasikaA073**
* Made the component for headerBar
* Made the component for headerBarMenu
* Made a component for SO logo
* tested with a sample carrousel

### CODE REVIEW

File: src/components/header/AppHeaderMenu.vue [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+</script>
+<template>
+
+
+  <!-- Create an array and render the items -->
+  <!-- Create a function and bind it to the onclick. Add logic to change active menu item  -->
+  <li class="nav-item" v-for="(item, index) in items" 
+  :key="index" 
+  :class="{ active: index == activeItem }"
+  :v-on:click="clickHandler(index)">
```
Comment (sandilsranasinghe) [inline src/components/header/AppHeaderMenu.vue:46]:
```suggestion
  v-on:click="clickHandler(index)">
```
no `:` before v-on
I would recommend using `@click` notation actually

```suggestion
  @click="clickHandler(index)">
```

File: src/components/header/AppHeaderMenu.vue [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          name: 'Prices',
+          url: '/prices'
+        },
+        {
+          name: 'Community',
+          url: '/community'
+        },
+        {
+          name: 'Support',
+          url: '/support'
```
Comment (sandilsranasinghe) [inline src/components/header/AppHeaderMenu.vue:21]: Shall we put some event related names here for now, like Home, About, Speakers, Contact

**Review by sandilsranasinghe — CHANGES_REQUESTED** [review]
In addition to the comments, it would be nice if you could use prettier to format the code

File: yarn.lock [hunk 3]
```diff
@@ -0,0 +1,962 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
[the remark this answers is not in the document]
Comment (SasikaA073) [reply yarn.lock:1]: I didn't get this latter part.
 I added yarn.lock to the .gitignore. Do I have to delete anything?

**Pull request closed without merging**